R/utils.R

```diff
@@ -308,6 +308,7 @@ set_make_local <- function(threads = FALSE,
   user_flags <- c()
   old_make_local_cmdstanr <- c()
   if (file.exists(make_local_path)) {
+    checkmate::assert_file_exists(make_local_path, access = 'rw')
```

Comment:
I added this here and below just as an extra safeguard to throw an error if we don't have read and write permissions for `make/local`

Reply:
Great!